.github/workflows/sync-repo-release.yml

```diff
@@ -0,0 +1,68 @@
+name: sync-repo-release
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "25 16 * * 6"
+
+jobs:
+  sync-gpustack:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Fetch latest release tag (include pre-release)
+        id: latest_release
+        run: |
+          tag=$(curl -s "https://api.github.com/repos/gpustack/gpustack/releases?per_page=1" | jq -r '.[0].tag_name')
+          echo "Latest release tag: $tag"
+          echo "tag=$tag" >> $GITHUB_OUTPUT
```

Comment:
How do we ensure we get the expected release, do you have reference about how github sort releases?

Reply:
This interface sorts by latest by default, and tests have shown that it works as expected.

Get the first page; there's only one per page, which is the latest one.